### ISSUE TITLE
UTF 8 characters in invoice editor window become html characters

### ISSUE THREAD

**Issue by orloxgr** [Dolibarr/dolibarr#16169]
# Bug
*Dolibarr invoice editor scrabbles Greek text for example γεια σου κοσμε becomes
`&gamma;&epsilon;&iota;&alpha; &sigma;&omicron;&upsilon; &kappa;&omicron;&sigma;&mu;&epsilon;.`
 If I enter γεια σου κοσμε in source mode and save the product in source code everything is working ok*

## Environment
- **Version**: [*13*]
- **OS**: [*centos 7*]
- **Web server**: [*apache*]
- **PHP**: [*7.4*]
- **Database**: [*maria*]

**Comment by orloxgr** [comment]
I managed to fix this by editing the file:
`/ htdocs / theme / eldy / ckeditor / config.js`
and adding this line:
config.entities_greek = false;

The issue was that this was passing to the invoices, so you had an invoice full of these characters instead of normal greek characters

**Comment by zuiko** [comment]
@orloxgr I fear that it will be the same with other non latin characters (cyrillic...)

**Comment by ksar-ksar** [comment]
It was with ODT template ?

**Comment by orloxgr** [comment]
> It was with ODT template ?

Yes, the export was on ODT template

**Comment by ksar-ksar** [comment]
Ok, then seems to be the same as https://github.com/Dolibarr/dolibarr/issues/16184

**Comment by eldy** [comment]
If you just enter ε into a product label and public note.
What do you have into database ? The ε or the &epsilon  ?

**Comment by orloxgr** [comment]
> If you just enter ε into a product label and public note.
> What do you have into database ? The ε or the &epsilon ?
After the change I made (https://github.com/Dolibarr/dolibarr/issues/16169#issuecomment-773522289) if I enter ε I get ε in database.
Predefined products are stored as html characters except if I edit them and resave them.
Example:
I put 1 predefined and an ε
![chrome_TLZCIdFjNu](https://user-images.githubusercontent.com/32058744/107487916-5e359f80-6b8f-11eb-95de-cfc22b48f525.png)
I edit the product and resave it
![chrome_Ld2U4KBuKw](https://user-images.githubusercontent.com/32058744/107487924-5ece3600-6b8f-11eb-8a5a-c8f75fe2946a.png)

This is a relative old dolibarr installation. Probably from dolibarr version 6. When 12 first came out I had this issue https://github.com/Dolibarr/dolibarr/issues/14245 but it was fixed I think at 12.3 maybe they are connected.
